test/OpenTelemetry.Tests/Trace/TracerTest.cs

```diff
@@ -74,6 +76,35 @@ public void Tracer_StartRootSpan_BadArgs_NullSpanName()
             Assert.Null(span3.Activity.DisplayName);
         }
 
+        [Fact(Skip = "See https://github.com/open-telemetry/opentelemetry-dotnet/issues/2803")]
+        public async Task Tracer_StartRootSpan_StartsNewTrace()
+        {
+            var exportedItems = new List<Activity>();
+
+            using var tracer = Sdk.CreateTracerProviderBuilder()
+                .AddSource("tracername")
+                .AddInMemoryExporter(exportedItems)
+                .Build();
+
+            async Task DoSomeAsyncWork()
+            {
+                await Task.Delay(10);
+                var newRootSpan = TracerProvider.Default.GetTracer("tracername").StartRootSpan("RootSpan2");
+                using (Tracer.WithSpan(newRootSpan))
```

Comment:
I think (not tested) that it can be simplified to 

```suggestion
                using (TracerProvider.Default.GetTracer("tracername").StartRootSpan("RootSpan2"))
```

Reply:
simplified